lib/address.js

```diff
@@ -635,7 +635,10 @@ Address.prototype.toCashBuffer = function() {
  */
 
 
-Address.prototype.toCashAddress = function() {
+Address.prototype.toCashAddress = function(withPrefix) {
+  if(withPrefix === undefined || typeof(withPrefix) !== 'boolean') {
```

Comment:
We want to throw if the argument is wrong type. It is better to complain than to silently accept something wrong.